package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "fusion-plugin-font-loader",
+  "version": "0.1.0",
+  "description": "The Fusion plugin for declaring and optimizing the loading of cutsom fonts",
+  "license": "MIT",
+  "repository": "fusionjs/fusion-plugin-font-loading",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/node.cjs.js",
+  "module": "./dist/node.es.js",
+  "browser": {
+    "./dist/node.cjs.js": "./dist/browser.es5.cjs.js",
+    "./dist/node.es.js": "./dist/browser.es5.es.js"
+  },
+  "es2015": {
+    "./dist/browser.es5.cjs.js": "./dist/browser.es2015.cjs.js",
+    "./dist/browser.es5.es.js": "./dist/browser.es2015.es.js"
+  },
+  "es2017": {
+    "./dist/browser.es5.cjs.js": "./dist/browser.es2017.cjs.js",
+    "./dist/browser.es5.es.js": "./dist/browser.es2017.es.js",
+    "./dist/browser.es2015.cjs.js": "./dist/browser.es2017.cjs.js",
+    "./dist/browser.es2015.es.js": "./dist/browser.es2017.es.js"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@babel/preset-react": "^7.0.0-beta.32",
+    "babel-eslint": "^8.0.2",
+    "create-universal-package": "^2.1.1",
+    "enzyme": "^3.1.1",
+    "enzyme-adapter-react-16": "^1.0.4",
+    "eslint": "^4.11.0",
+    "eslint-config-fusion": "^0.1.2",
+    "eslint-plugin-cup": "^1.0.0",
+    "eslint-plugin-flowtype": "^2.39.1",
+    "eslint-plugin-prettier": "^2.3.1",
+    "eslint-plugin-react": "^7.4.0",
+    "flow-bin": "^0.59.0",
+    "fusion-core": "^0.2.3",
+    "nyc": "^11.3.0",
+    "prettier": "1.8.2",
+    "react": "^16.1.1",
+    "react-dom": "^16.1.1",
+    "tape-cup": "^4.7.1",
+    "unitest": "^1.1.0"
+  },
+  "peerDependencies": {
+    "fusion-core": "^0.2.3",
+    "react": "14.x - 16.x"
```

Comment:
If this depends on react, should it be called `fusion-plugin-font-loader-react`? or something?

Reply:
It exports both a plugin and an HOC. Kind of resistant to the longer name

Reply:
I agree the long names become a bit rediculous. I'm just concerned about confusion once we support `preact`, `inferno`, etc. If someone sees this plugin, how would they know it is coupled with react?

Reply:
renamed